tests/cross/src/main/scala/tests/BaseCompletionSuite.scala

```diff
@@ -101,6 +103,52 @@ abstract class BaseCompletionSuite extends BasePCSuite {
     }
   }
 
+  def checkEdit(
```

Comment:
This signature is so complicated it’s probably better to keep adding more parameters instead of duplicating the method. What do you think?

Reply:
yeah, I agree. I was uneasy too with the amount of duplication